irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -15,6 +15,8 @@
  * limitations under the License.
  */
 
+#include <boost/format.hpp>
```

Comment:
Should be below as for our [style](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
Yes